### ISSUE TITLE
Number of iterations?

### ISSUE THREAD

**Issue by kyleabeauchamp** [choderalab/yank#131]
Should it be possible to adjust the number of iterations after setup?  The command line inputs suggest it's allowed, but it doesn't seem to work:

```
kyleb@src:~/src/choderalab/yank/examples/host-guest-implicit$ yank run --store=output --verbose --iterations=600
[...]

Iteration 812 / 1000
```

This was allegedly fixed by #125, but I'm still having issues.

**Comment by jchodera** [comment]
I think this is fixed now. Reopen if not.

**Comment by pgrinaway** [comment]
I'm checking right now

**Comment by pgrinaway** [comment]
Ok, so if I do:
```
yank setup binding amber --setupdir=setup --ligname=MOL --store=output --iterations=1000 \
--restraints=harmonic --gbsa=OBC2 --temperature=300*kelvin --verbose

yank run --iterations=800 --store=output --verbose
```

It indicates that it's running 1000 iterations. We can either just change the docs to have the setup set the number of iterations or figure out what's going on here.

**Comment by pgrinaway** [comment]
(also, I just cloned the repo 20 min ago)

**Comment by pgrinaway** [comment]
Ok, I had thought my example didn't work because I specified a smaller number of iterations in the `yank run` command, but after trying this:

```
yank setup binding amber --setupdir=setup --ligname=MOL --iterations=10 --store=output \ 
--restraints=harmonic --gbsa=OBC2 --temperature=300*kelvin --verbose

yank run --iterations=80 --store=output --verbose
```

It only runs 10 iterations.

**Comment by jchodera** [comment]
Addressed by #195.
